pkg/session_recording/recordings.jsx

```diff
@@ -222,13 +233,31 @@
             cockpit.location.go([recording.id]);
         }
 
+        /*
+         * Handles Datepicker Since.
+         */
+        handleDateSinceChange(date) {
+            this.setState({dateSince: date});
+            this.props.onDateSinceChange(date);
+        }
+
+        /*
+         * Handles Datepicker Until.
+         */
+        handleDateUntilChange(date) {
+            this.setState({dateUntil: date});
+            this.props.onDateUntilChange(date);
+        }
+
         render() {
-            var columnTitles = [_("User"), _("Start"), _("End"), _("Duration")];
-            var list = this.props.list;
-            var rows = [];
-            for (var i = 0; i < list.length; i++) {
-                var r = list[i];
-                var columns = [r.user,
+            const dateSince = this.state.dateSince;
+            const dateUntil = this.state.dateUntil;
+            let columnTitles = [_("User"), _("Start"), _("End"), _("Duration")];
+            let list = this.props.list;
+            let rows = [];
+            for (let i = 0; i < list.length; i++) {
+                let r = list[i];
+                let columns = [r.user,
```

Comment:
Basically, in ES6 it's considered to use let always when you don't need var. This will lock let to scope without possible conflicts with variables on upper levels ( possible problems with closures ). Also, it's more memory efficient - cause let will be destroyed immediately when scope is dead.